FV3/wrapper/Makefile

```diff
@@ -31,12 +31,7 @@ export PRINT_HELP_PYSCRIPT
 
 BROWSER := python3 -c "$$BROWSER_PYSCRIPT"
 
-# This must be updated if the tag used in docker/Makefile is changed
-DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu7-mpich314-nocuda
-BUILD_FROM_INTERMEDIATE=y
-
-PYTHON_FILES = $(shell git ls-files | grep -e 'py$$' | grep -v -e '__init__.py' -e 'ipcluster_config.py')
-PYTHON_INIT_FILES = $(shell git ls-files | grep '__init__.py')
+# This must be updated if the tag used in Makefile is changed
```

Comment:
Is this comment obsolete/deletable? There's no more docker image and so no more tag.